crates/bevy_ui/src/focus.rs

```diff
@@ -53,6 +53,27 @@ impl Default for Interaction {
     }
 }
 
+/// Describes whether [`Interaction`] component should remain in the `Clicked` state after
```

Comment:
Yes, I think we should rename it to `Interaction::Pressed` so that it makes more sense when we eventually start supporting `Interaction::Released` (#5769)

Reply:
Or, hmm. On second thought no. 

Lets keep the name as `Interaction::Clicked` and do the rename in #7240 so that those two changes are kept together. Will be easier to review that way.